src/screens/Podcast/components/EpisodeItem/episodeItem.tsx

```diff
@@ -1,22 +1,18 @@
 import React from 'react'
 import { View, TouchableOpacity } from 'react-native'
+import { Episode } from 'src/common/types'
+import { INCREASE_VALUE_FOR_LIST } from './common/constants'
+import { getFormattedDate, DateFormatType } from 'src/helpers'
 import PlayIcon from 'src/assets/images/playEpisode.svg'
 import { Heading, HeadingType, PlainText } from 'src/components'
 import styles from './styles'
 
-type Episode = {
-  name: string
-  createdAt: string
-}
-
 type Props = {
   number: number
```

Comment:
`number: number` is not clear for me. Let's name it, for example, `position: number`